WORKSPACE

```diff
@@ -249,6 +250,14 @@ load("//build/bazelutil:seed_yarn_cache.bzl", "seed_yarn_cache")
 
 seed_yarn_cache(name = "yarn_cache")
 
+yarn_install(
+    name = "npm_mirror_npm",
+    package_json = "//pkg/cmd/mirror/npm:package.json",
+    yarn_lock = "//pkg/cmd/mirror/npm:yarn.lock",
+    symlink_node_modules = True,
+    strict_visibility = False,
+)
```

Comment:
Just calling out that this is an *online* installation from `yarnpkg.com` for now. The next PR in the series will switch the lockfile so that this downloads from our mirror.

Reply:
IIUC nothing that's downloaded here will actually make it into the `cockroach` build, so this should be fine.

Reply:
Correct! This is only used for parsing and regenerating `yarn.lock` files that determine where to download our actual dependencies. Since those files are checked in (and will be switching to our mirror shortly), I think I'm okay with this temporarily.